bluebrain/deployment/environments/applications_hpc.yaml

```diff
@@ -32,20 +32,20 @@ spack:
           +ngv+metabolism: '{name}-multiscale/{version}'
           ^petsc+complex: '{name}-complex/{version}'
   specs:
-    - neuron+tests+coreneuron
```

Comment:
maybe not necessary to remove teh `+coreneuron` here. It doesn't hurt and at the same time it avoid confusion to keep it explicit.

Reply:
I see your point.. I reverted these changes